src/lib/builders/tags-input/index.ts

```diff
@@ -22,15 +31,34 @@ export type CreateTagsInputArgs = {
 	// Whether the tags should be unique
 	unique?: boolean;
 	// What to do on blur
-	// blur?: 'nothing' | 'add' | 'clear';
+	blur?: 'nothing' | 'add' | 'clear';
+	// Whether the tag should be added on paste
+	addOnPaste?: boolean;
```

Comment:
```suggestion
	allowPaste?: boolean;
```

Reply:
allowPaste is too vague imo. 

I copied addOnPaste from zag js. The idea being that it will automatically add a tag on paste (if valid)

I’ll change it though if you want

Reply:
> The idea being that it will automatically add a tag on paste (if valid)
Oh okay! We can leave it as is then.